lib/cartopy/crs.py

```diff
@@ -2493,6 +2493,33 @@ def transform_points(self, src_crs, x, y, z=None, trap=False):
         return result
 
 
+class WinkelTripel(_WarpedRectangularProjection):
```

Comment:
Do we need to do any of the nan handling here like we are doing in the Robinson projection? i.e. should this subclass `Robinson` instead if it is similar to that?

Reply:
I was able to generate maps containing maps with no problem. Are there other tests for nans that I would need to do?